### PR TITLE
(GH-747) add extension to the licensefile

### LINKED ISSUE #747: Chocolatey package license file extension
The Chocolatey package currently has no extension on the LICENSE file. Without an extension the file is not viewable on the website.

image

### PR DESCRIPTION
**Pull request by mkevenaar**
Merging this fixes #747

### CODE REVIEW

File: nuspec/chocolatey/ChocolateyGUI.nuspec [hunk 1]
```diff
@@ -52,7 +52,7 @@ All release notes for Chocolatey GUI can be found on the GitHub site - https://g
     <file src="chocolateyInstall.ps1" target="tools"/>
     <file src="chocolateyUninstall.ps1" target="tools"/>
     <file src="..\..\BuildArtifacts\ChocolateyGUI.msi" target="tools"/>
-    <file src="..\..\LICENSE" target="tools\LICENSE"/>
+    <file src="..\..\LICENSE" target="tools\LICENSE.txt"/>
```
Comment (AdmiringWorm) [inline nuspec/chocolatey/ChocolateyGUI.nuspec:55]: This change won't result in what you're trying to achieve.

When the last segment don't match the actual name of the imported file, then a directory will be created.

image

Comment (mkevenaar) [reply nuspec/chocolatey/ChocolateyGUI.nuspec:55]: TIL!

@AdmiringWorm @gep13 do we want to rename the LICENSE file inside the repo to LICENSE.txt or do we want to copy the file over into the tools folder with the correct extension?

Comment (gep13) [reply nuspec/chocolatey/ChocolateyGUI.nuspec:55]: I have no objections to renaming the actual file to include the extension.

Comment (mkevenaar) [reply nuspec/chocolatey/ChocolateyGUI.nuspec:55]: I’ll try and pick that up later today or tonight.

Comment (mkevenaar) [reply nuspec/chocolatey/ChocolateyGUI.nuspec]: The file have been renamed. All other locations pointing to the license file have been updated as well.

**Comment by gep13** [comment]
@mkevenaar thanks for doing this!

**Pull request merged**